### ISSUE TITLE
Multi Account bot Proxy

### ISSUE THREAD

**Issue by tokar86a** [PokemonGoF/PokemonGo-Bot#6230]
### Expected Behavior
That the bot should start

### Actual Behavior
I get this error message

https://pastebin.com/0z3dCXrY 

### Output when issue occurred
https://pastebin.com/0z3dCXrY

### Steps to Reproduce
Put proxys in the MultiBotConfigProxy.txt file

### Other Information
OS: 
 
Branch: Master
 
Git Commit: 
 
Python Version: 
 
Any other relevant files/configs (eg: path files)

**Comment by MerlionRock** [comment]
I am testing something.... and hopefully will solve this issue. What OS are you using? Mac or Windows?

**Comment by tokar86a** [comment]
Windows

**Comment by MerlionRock** [comment]
@tokar86a go to discord, I posted a file in support English channel, help me test if it solve your problem.

**Comment by tokar86a** [comment]
2017-10-24 18:55:20 Thread-0 using account po7610
2017-10-24 18:55:20 1Z9G8C5Y2K is valid
'export' is not recognized as an internal or external command,
operable program or batch file.

**Comment by MerlionRock** [comment]
@tokar86a Yup, I make a mistake for Windows. Export command is not for windows. Fixing....
